File: QuanLyHienMauApi/QuanLyHienMauApi/Models/CoSoYTe.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace QuanLyHienMauApi.Models
{
    public class CoSoYTe
    {
        [Key]
        [ForeignKey("ThongTinTaiKhoan")]
        public int ID { get; set; }

        public string? MaSoThue { get; set; }

        public string? AnhGiayPhep { get; set; }

        public string? AnhQuyetDinh { get; set; }

        public int TrangThaiDuyet { get; set; } = 0;

        // Liên kết ngược lại bảng NguoiDung
        public virtual NguoiDung? ThongTinTaiKhoan { get; set; }
    }
}

File: HienMau/app/src/main/java/com/example/hienmau/models/CoSoYTe.java
package com.example.hienmau.models;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class CoSoYTe implements Serializable {
    @SerializedName("ID")
    public int id;
    @SerializedName("MaSoThue")
    public String maSoThue;
    @SerializedName("TrangThaiDuyet")
    public int trangThaiDuyet;

    @SerializedName("ThongTinTaiKhoan")
    public NguoiDung thongTinTaiKhoan;
}
